projects/train/train/data/supervised/frequency_domain.py

```diff
@@ -1,11 +1,12 @@
 import torch
 
+from ml4gw.spectral import truncate_inverse_power_spectrum
 from ml4gw.transforms.qtransform import SingleQTransform
 from train.data.base import Tensor
 from train.data.supervised.supervised import SupervisedAframeDataset
 
 
-class SpectrogramSupervisedAframeDataset(SupervisedAframeDataset):
+class SpectrogramDomainSupervisedAframeDataset(SupervisedAframeDataset):
```

Comment:
Not relevant to this PR, but we could consider removing `Aframe` from all these names to make them a little more manageable

Reply:
Good idea